kubernetes/resources_api/overlays/staging/database-service.yaml

```diff
@@ -4,4 +4,4 @@ metadata:
   name: resources-postgres
 spec:
   type: ExternalName
-  externalName: python-staging.czwauqf3tjaz.us-east-2.rds.amazonaws.com
+  externalName: python-staging-aurora.cluster-czwauqf3tjaz.us-east-2.rds.amazonaws.com
```

Comment:
ah crap, of course I have a similar change sitting uncommitted 🤦  

We're likely never going to use the aurora cluster.  The current staging database is at `staging-postgres.pgo.svc.cluster.local`  or `staging-postgres.pgo` for short.

Reply:
while you're in here, could you please also update the reference here: https://github.com/OperationCode/operationcode_infra/blob/main/kubernetes/operationcode_python_backend/overlays/staging/deployment.yaml#L13